docs/custom_mutators.md

```diff
@@ -132,13 +136,21 @@ def deinit():  # optional for Python
     for a specific queue entry, use this function. This function is most useful
     if `AFL_CUSTOM_MUTATOR_ONLY` is **not** used.
 
+- `splice_optout` (optional):
```

Comment:
How can this possibly work in python? Wouldn't the fn always be available?

Reply:
why would the function be available when it is not in the python script?  I dont know python really, but logically it does not make sense to me :)

Reply:
Do you check somewhere in the code if it's set in python?

Reply:
fixed